### PR TITLE
Tapping views/visitors/likes/comments in Insights switches to Days summary type

### LINKED ISSUE #261: Make each section of Today in Insights switch graph selected row
When tapping one of the following sections:

Switch to the similarly labeled section on the graph after the view is flipped to "Days".

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #261 

When tapping the views/visitors/likes/comments buttons in Insights, the view is now switched to the graph, selects Days and then changes the summary type to the appropriate one according to what tapped.

## What to Test
1. Selection of each item in Insights. Make sure it switches to Days and selects the right summary type.
2. Switch back and forth between periods and selected bars in the graph, do #1 again and make sure it works.
3. Try to break it.

## How to Test
Use StatsDemo built today (18-JUN) - it includes this PR and the other two dependent PRs still open.

Needs Review: @jleandroperez, @isaackeyet

### CODE REVIEW

**Comment by astralbodies** [comment]
cc: @daniloercoli

File: WordPressCom-Stats-iOS/InsightsTableViewController.m [hunk 1]
```diff
@@ -208,16 +130,16 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 {
     [tableView deselectRowAtIndexPath:indexPath animated:YES];
     
-    if (indexPath.section == 2) {
-        if ([self.statsTypeSelectionDelegate conformsToProtocol:@protocol(WPStatsTypeSelectionDelegate)]) {
-            [self.statsTypeSelectionDelegate viewController:self changeStatsTypeSelection:StatsTypeDays];
+    if (indexPath.section == 2 && indexPath.row == 0) {
```
Comment (jleandroperez) [inline WordPressCom-Stats-iOS/InsightsTableViewController.m]: Cocococonstants! :smile:

File: WordPressCom-Stats-iOS/InsightsTableViewController.m [hunk 2]
```diff
@@ -308,4 +226,216 @@ - (void)setupRefreshControl
...
+    [viewsText insertAttributedString:[[NSAttributedString alloc] initWithString:@" "] atIndex:1];
+    [viewsText appendAttributedString:[[NSAttributedString alloc] initWithString:@"  "]];
+    [viewsText addAttribute:NSForegroundColorAttributeName value:[WPStyleGuide greyDarken20] range:NSMakeRange(0, viewsText.length)];
+
+    return viewsText;
+}
+
+- (NSMutableAttributedString *)visitorsAttributedString
+{
+
```
Comment (jleandroperez) [inline WordPressCom-Stats-iOS/InsightsTableViewController.m]: Nipticky: extra `\n` here

**Comment by jleandroperez** [comment]
@astralbodies the feature looks great!!. I've been unable to break it, so far.

:shipit: on my side!

File: WordPressCom-Stats-iOS/InsightsTableViewController.m (hunk 1 above)
Comment (astralbodies) [inline WordPressCom-Stats-iOS/InsightsTableViewController.m]: I KNOW I KNOW

**Pull request merged**